.github/workflows/pr-cleanup.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
           fi
 
       - name: Delete image
+        continue-on-error: true
```

Comment:
I feel like the following steps should also have continue-on-error, right?

Reply:
They will not fail. And every step is independent and only depends on the outcome of the previous step.